package.json

```diff
@@ -6,9 +6,16 @@
     "start": "next start"
   },
   "dependencies": {
+    "@mux/mux-node": "^7.3.0",
     "@mux/mux-player-react": "^1.8.0",
+    "@mux/mux-uploader-react": "^0.1.0-canary.218-5856b1b",
     "next": "^13.2.4",
     "react": "^18.2.0",
     "react-dom": "^18.2.0"
+  },
+  "devDependencies": {
+    "@types/node": "18.15.11",
+    "@types/react": "18.0.33",
+    "typescript": "5.0.3"
```

Comment:
The `typescript` dependency should be removed and `npm install` should be re-run.

Reply:
👍 Removed it and uninstalled.